src/sqlfluff/core/config.py

```diff
@@ -144,6 +144,34 @@ def get_global(cls) -> "ConfigLoader":
             global_loader = cls()
         return global_loader
 
+    @staticmethod
+    def _walk_toml(config: Dict[str, Any], base_key=()):
+        """Recursively walk the nested config inside a TOML file."""
+        buff: List[tuple] = []
+        for k, v in config.items():
+            if isinstance(v, dict) and k.endswith("__options"):
+                key = base_key + (k[:-9],)
+                buff.extend(ConfigLoader._walk_toml(v, key))
+            else:
+                key = (base_key or ("core",)) + (k,)
+                buff.append((key, v))
+
+        return buff
+
+    @staticmethod
+    def _get_config_elems_from_toml(fpath: str) -> List[Tuple[tuple, Any]]:
+        """Load a config from a TOML file and return a list of tuples.
+
+        The return value is a list of tuples, were each tuple has two elements,
+        the first is a tuple of paths, the second is the value at that path.
+        """
+        import toml
```

Comment:
Could this import be moved to the top of the file? That's generally the preferred location for imports.

Reply:
Sure. I'll do that.